client/src/components/Login/index.js

```diff
@@ -45,12 +102,14 @@ class Login extends Component {
   handleSubmit = event => {
     const { history } = this.props;
     const { username, password } = this.state;
+    console.log("info", username, password);
     event.preventDefault();
     loginHelper({ username, password })
       .then(result => {
         if (result.logged) {
           history.push("/cars");
         } else {
+          console.log("error");
           this.setState({
```

Comment:
Remove the `console.log` in line L96, L105 and L112

Reply:
Done and ready to merge c: